### PR TITLE
OSDOCS#11121: [OLMv1] Remove references to RukPak and bundle deployments

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.17

Issue: [OSDOCS-11121](https://issues.redhat.com//browse/OSDOCS-11121)

Link to docs preview:
* https://82372--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/ce/managing-ce.html
* https://82372--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/understanding/olm-packaging-format.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: operators/understanding/olm-packaging-format.adoc [hunk 1]
```diff
@@ -75,13 +75,11 @@ include::modules/olm-fb-catalogs-automation.adoc[leveloffset=+2]
...
+
 [role="_additional-resources"]
 .Additional resources
 
 * xref:../../operators/olm_v1/index.adoc#olmv1-about[About {olmv1} (Technology Preview)]
 * xref:../../operators/understanding/olm-packaging-format.adoc#olm-bundle-format_olm-packaging-format[{olmv0-caps} bundle format]
 
-include::modules/olm-rukpak-provisioner.adoc[leveloffset=+2]
-include::modules/olm-rukpak-bd.adoc[leveloffset=+2]
 endif::openshift-dedicated,openshift-rosa[]
```
Comment (michaelryanpeter) [inline operators/understanding/olm-packaging-format.adoc:85]: @adellape Should I just cut this whole part?

**Review by everettraven — APPROVED** [review]

**Comment by michaelryanpeter** [comment]
@jianzhangbjz or @kuiwang02 PTAL

**Review by joelanford — APPROVED** [review]

**Comment by Xia-Zhao-rh** [comment]
LGTM

**Review by ShaunaDiaz — COMMENTED** [review]
The PR looks good, but in a quick scroll through looking for content you don't want there, I did see an unresolved snippet include (include:snippets/olmv1-tp-extension-support.adoc[]) in Step 3 of this procedure:
 https://82372--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/ce/managing-ce.html#olmv1-finding-operators-to-install_managing-ce

**Comment by michaelryanpeter** [comment]
Good catch @ShaunaDiaz. TY!

**Pull request merged**